angular/projects/admin-nrpti/src/app/records/inspections/inspection-detail/inspection-detail.component.html

```diff
@@ -0,0 +1,100 @@
+<main class="detail-page">
+  <div>
+    <section class="content-section">
+      <div class="content-grid">
+        <div class="grid-section-header">
+          <div class="grid-section-header-title">
+            <span class="grid-section-header-title-text">Shared Data [Master]</span>
+            <span class="grid-section-header-title-sub-text">
+              Last Edited {{ (data && data._master.dateUpdated | date: 'mediumDate') || '-' }}
+            </span>
+          </div>
+          <div class="grid-section-header-button">
```

Comment:
Remove this entirely if there is no edit functionality.  Same goes for the Order.

Reply:
Commented out for now while its not functional.